src/domains/gps-common/03_storage_account.tf

```diff
@@ -3,37 +3,44 @@
 ##################
 
 # storage
-module "payments_receipt_sa" {
+module "gpd_archive_sa" {
   source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//storage_account?ref=v7.18.0"
 
-  name                            = replace(format("%s-payments-sa", local.project), "-", "")
+  name                            = replace(format("%s-archive-sa", local.project), "-", "")
   account_kind                    = "StorageV2"
   account_tier                    = "Standard"
   account_replication_type        = var.storage_account_replication_type
```

Comment:
there isn't a dedicated variables for `storage_account_replication_type`. If doesn't exist archive policy It could be `GZRS`